### PR TITLE
Create hidden-lake.yml

### PR DESCRIPTION
**Pull request by number571**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by kokomo123** [comment]
Hi there, can you switch all the vs to "x" so the checkmark can be attributed to the tasks? Thank you!

**Comment by number571** [comment]
Updated

File: software/hidden-lake.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+licenses:
+  - MIT
+platforms:
+  - Go
+tags:
+  - Communication - Custom Communication Systems
+source_code_url: https://github.com/number571/go-peer/tree/master/cmd/hidden_lake
+stargazers_count: 202
+updated_at: '2024-06-06'
+archived: false
```
Comment (kokomo123) [inline software/hidden-lake.yml:13]:
```suggestion
```
One more thing, these will be automatically filled in so no need for these lines!

**Comment by nodiscc** [comment]
While this is interesting and novel, I don't think it is at the level of end-user-friendliness expected in awesome-selfhosted:

> The Hidden Lake network can be used as a communication platform for applications that are heterogeneous in nature. This is made possible by the go-peer protocol, which does not define any application use. As a result, you can create your own applications at several levels: either at the go-peer library level or at the HL services level ([example](https://github.com/number571/go-peer/tree/master/cmd/secpy_chat)),

https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues/594#issuecomment-2381289295:

> [...] does not qualify:   requires you to write application code before producing a working end-user application

You'll notice that other anonymizing networks such as i2p are not listed here either.

An end-user application running on top of Hidden Lake would probably be accepted.

Closing, feel free to reopen.

**Pull request closed without merging**